src/repository.rs

```diff
@@ -30,6 +30,11 @@ pub struct Repo {
     forks_count: u64,
     archived: bool,
     disabled: bool,
+    has_issues: bool,
+    has_projects: bool,
+    has_wiki: bool,
```

Comment:
The build has failed for this PR. `has_issues` and `has_wiki` are currently already defined in `master` (added by #17).
Please fix this, and if the build succeeds, I'll merge the PR :slightly_smiling_face: